src/openrct2/core/Http.cURL.cpp

```diff
@@ -141,7 +141,9 @@ namespace Http
         CURLcode curl_code = curl_easy_perform(curl);
         if (curl_code != CURLE_OK)
         {
-            throw std::runtime_error("Failed to perform request");
+            throw std::runtime_error(
+                u8"Failed to perform request. curl error code: " + std::to_string(curl_code) + ": "
```

Comment:
Hmm, is the `u8` correct here? I wanted to have it result in `std::string` so I can concatenate with `+`, but I think it's not `u8`…

Reply:
It was `std::literals` and `""s`